### ISSUE TITLE
Change some amenities colour to grey

### ISSUE THREAD

**Issue by Tomasz-W** [gravitystorm/openstreetmap-carto#3326]
I propose to change colour of some non-cultural, non-service, free standing objects from amenity-brown to man-made-grey. Amenity-brown is heavy loaded right now and it includes tags from many different categories. As we already render towers, windmills or crosses in man-made-gray, and will render tourist information objects too, I think we should use this colour also for:
* `amenity=bench`
* `leisure=picnic_table`
* `amenity=waste_basket`
* `amenity=waste_disposal`
* `amenity=recycling`

**Comment by kocio-pl** [comment]
At the moment the difference is more or less whether the objects are usable (brown) or are mainly landmarks (gray), but that might make sense to "demote" some less important, but usable objects to gray to just offload brown.

**Comment by Adamant36** [comment]
We have a lot of cement bench's and picnic tables where I live that know one really uses and serve more like landmarks or markers for interesting places. So I sort of like this idea.

**Comment by kocio-pl** [comment]
It's possible to do it, but at the moment I'm not focused on this problem.

**Comment by Adamant36** [comment]
@Tomasz-W, as it currently stands both the general amenity=recycling and recycling_type=centre is rendered the same way and the code is connected. Recycling_type=container is not rendered at all though. What I think we should do is that recycling_type=centre should remain as amenity brown and maybe rendering its area could happen also. Then add rendering for recycling_type=container in grey. Which might spur mappers to use the recycling type tags more. Plus, also leave the general amenity=recycling as amenity brown but make it a separate feature in the code from the other two. What do you think?

**Comment by Adamant36** [comment]
Some test renders of everything except amenity=recycling and amenity=shelter
![benches and waste baskets in park](https://user-images.githubusercontent.com/30259065/45605403-9dc52500-b9f1-11e8-933f-622bc6d880dc.png)
![picnic tables park](https://user-images.githubusercontent.com/30259065/45605406-a289d900-b9f1-11e8-9c2c-10ae2ce35b2c.png)
![picnic tables retail area](https://user-images.githubusercontent.com/30259065/45605408-a4ec3300-b9f1-11e8-84e1-0c76fa5e5070.png)
![waste disposel grass](https://user-images.githubusercontent.com/30259065/45605410-a74e8d00-b9f1-11e8-888d-8dc022c0f3e5.png)
![waste disposel residential](https://user-images.githubusercontent.com/30259065/45605412-a87fba00-b9f1-11e8-9850-fb0d0d67cf98.png)

**Comment by Adamant36** [comment]
amenity=recycling@kocio-pl good point. mostly the issue is that @tomasz-W wanted to render amenity=recycling in gray, but I think recycling centers are amenities and should stay brown. Recycling containers could probably be colored gray, since they aren't amenities per say, but they aren't currently rendered. I think it would be worth showing them. Especially since we are already rendering normal waste baskets. 

I thought doing that and rendering them in grey color would be a good middle ground to keeping amenity=recycling as brown. Then as a side thing, the numbers for recycling centers might increase if they are rendered as areas. 

As it is the code for them currently looks like this 
 `` [feature = 'amenity_recycling'][recycling_type = 'centre'][zoom >= 17],
  [feature = 'amenity_recycling'][zoom >= 19] {
    marker-file: url('symbols/amenity/recycling.svg');
    marker-fill: @amenity-brown;
    marker-placement: interior;
    marker-clip: false;
    [access != ''][access != 'permissive'][access != 'yes'] {
      marker-opacity: 0.33;``

If recycling containers was added to the map in a different color then the other two, it would have to be codded separately. @Tomasz-W never said if he wanted amenity=recycling rendered in grey or if recycling centers were included. I was pointing out that they share the same code and it would have to be separated if they are rendered differently. Although, I guess it doesn't really matter. Hopefully that is a clearer explanation though at least.

**Comment by kocio-pl** [comment]
GitHub hint - if you want to show a block of code, try using this notation:

`https://github.com/gravitystorm/openstreetmap-carto/blob/90f769d57eff2ca55a297a7c4952e2206ca8e1f0/amenity-points.mss#L660-L669`

which gives the output like this:

https://github.com/gravitystorm/openstreetmap-carto/blob/90f769d57eff2ca55a297a7c4952e2206ca8e1f0/amenity-points.mss#L660-L669

**Comment by Adamant36** [comment]
Thanks. I was wondering how to do that. It looks a lot better.

**Comment by Tomasz-W** [comment]
Yeah, containers/ centres may be distinguished from each other by man-made grey for the first one and amenity-brown for second one.

Generally I like the effect from test renderings. The difference is minimal, but it's more logical in this way. @Adamant36 Why did you exclude amenity=shelter?

**Comment by polarbearing** [comment]
Generally in favour of this move. For different types of recycling we would need to see a comparison, if the same icon is used for different meaning the difference should be sufficiently recognisable. 

BTW, what is the big beast in the middle of this tile (man_made grey)?
<img src=https://user-images.githubusercontent.com/30259065/45605410-a74e8d00-b9f1-11e8-888d-8dc022c0f3e5.png>

**Comment by Adamant36** [comment]
Its either man_made=storage_tank, or Man_made=silo. It just got merged a few commits back. The icon is rather large, but they are large figures anyway.

**Comment by Adamant36** [comment]
@Tomasz-W, sorry about that. I was planning on including shelters and I spaced it until I was already working on something else. Here's some tests of it though. 
![shelter](https://user-images.githubusercontent.com/30259065/45609042-0371db00-ba0b-11e8-85de-54655f589b37.png)
![shelter 2](https://user-images.githubusercontent.com/30259065/45609045-053b9e80-ba0b-11e8-9df4-c7d75da16486.png)
I think the test renders for everything look good and it makes more intuitive sense for some reason. Would you be willing to come up with a new icon for recycling containers if need be? Like @kocio-pl says, just doing them in a different color might not be  clear enough.

**Comment by Tomasz-W** [comment]
We have been trying with container designs some time ago, and it didn't go readable (see: https://github.com/gravitystorm/openstreetmap-carto/issues/419#issuecomment-340256807). I would stay just with colour change, as all amenity=recycling types are rendered the same way at the moment, it's a step forward eitherway.

**Comment by Adamant36** [comment]
Recycling containers in grey. I think it works. 
![recycling container](https://user-images.githubusercontent.com/30259065/45610686-b0e7ed00-ba11-11e8-89d0-7cbb3eb79a0e.png)
![recycling container 2](https://user-images.githubusercontent.com/30259065/45610689-b47b7400-ba11-11e8-9bb0-b1ef38f00f96.png)

**Comment by Tomasz-W** [comment]
Just for the note: at the moment recycling containers and recycling centres are rendered the same. There is PR needed to render containers in man-made-grey and centres in amenity-brown.

**Comment by matthijsmelissen** [comment]
@Adamant36 I think we should add memorials to the list too. What do you think?

**Comment by Adamant36** [comment]
Sounds like a good idea. I think they would qualify.

**Comment by Tomasz-W** [comment]
You mean change memorials colour to grey? They are brown because they have cultural/ historical meaning (see https://github.com/gravitystorm/openstreetmap-carto/issues/3395), so I'm very against changing them to grey as trash bins etc. are.

**Comment by jeisenbe** [comment]
The only remaining item on the list is `amenity=recycling`, otherwise this issue can be closed.
